### PR TITLE
[ROCm] Creating GpuLaunchKernel

### PR DESCRIPTION
**Pull request by jerryyin**
This PR is a follow-up to the original PR #28343. The reviewer requested to break down the original large PR to a series of smaller ones. According to the plan here, this PR is the fifth one in the whole series.

@chsigg @whchung

### CODE REVIEW

File: tensorflow/core/util/gpu_kernel_helper.h [hunk 1]
```diff
@@ -63,6 +63,39 @@ using gpuError_t = hipError_t;
 #define GetGPUStream(context) context->eigen_gpu_device().stream()
 
 namespace tensorflow {
+// Launches a GPU kernel through cudaLaunchKernel in Cuda environment, or
```
Comment (chsigg) [inline tensorflow/core/util/gpu_kernel_helper.h]: CUDA instead of Cuda, but that shouldn't gate this PR.

**Review by chsigg — DISMISSED** [review]

File: tensorflow/core/util/gpu_kernel_helper.h (hunk 1 above)
Comment (jerryyin) [reply tensorflow/core/util/gpu_kernel_helper.h]: Appreciate this. Will update this as a follow-up PR mentioned in our conversation [here](https://github.com/tensorflow/tensorflow/pull/28568#discussion_r283381197)

Comment (jerryyin) [reply tensorflow/core/util/gpu_kernel_helper.h]: Can't beat the speed of a newly commited kernel `tridiagonal_matmul_op_gpu.cu.cc`. Did a force push to address the renaming here. As well as updating `Cuda` to `CUDA`. Now should be good.

This update serves as a good example to make me understand why a renaming needs to be updated in two phases.

**Review by chsigg — DISMISSED** [review]

**Comment by jerryyin** [comment]
It looks like this PR is already merged [here](https://github.com/tensorflow/tensorflow/commit/921ee23536ae07286478b0ce0180ad816d0b3b63), but github is not showing it correctly. I'm not entirely sure what repercussion will have if I close PR, so will just leave it. FYI, @chsigg @whchung,

**Comment by whchung** [comment]
@jerryyin you should be able to close this PR without causing trouble on the mainline.

**Pull request closed without merging**